p2plending/frontend/src/components/BrowseByLanguage.js

```diff
@@ -22,6 +22,7 @@ class BrowseByLanguage extends Component {
     
     render() {
     const { languages } = this.state;
+    console.log(this.state);
```

Comment:
can you remove this console.log? It's generally bad practice to have debugging console logs pushed.

Reply:
Done.